batch/batch.go

```diff
@@ -23,8 +23,11 @@ import (
 	"time"
 )
 
+// DefaultWaitInterval is the default WaitInterval for Func.
+var DefaultWaitInterval = 500 * time.Microsecond
+
 // DefaultMaxDuration is the default MaxDuration for Func.
-var DefaultMaxDuration = 200 * time.Microsecond
+var DefaultMaxDuration = 10 * time.Millisecond
```

Comment:
Makes sense. Maybe as a future change we could make the sqlgen batcher options configurable from outside of the package so that it's easier to tweak this without changing thunder.